ceph/ceph_admin/bootstrap.py

```diff
@@ -364,7 +365,7 @@ def bootstrap(self: CephAdmProtocol, config: Dict):
                 "snmp_gateway",
                 "loki",
             ]
-            if _rhcs_version >= 7.0:
+            if self.cluster.rhcs_version >= LooseVersion("7.0"):
```

Comment:
why `self.cluster.rhcs_version` ? `_rhcs_version` passed through the upgrade suite which needs to be validated.

Reply:
Based on the --rhbuild or rhcs_version it gets assigned to this variable ultimately
https://github.com/AmarnatReddy/cephci/blob/e947fcb256a797cba221d44b7299205b8a753871/ceph/ceph_admin/bootstrap.py#L224

Logs without RHCS Version : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-4ISGD8/

Reply:
@AmarnatReddy as per my understanding `_rhcs_version` is pre-upgrade version and `rhBuild` is post upgrade version. How could both same ?
Refer - https://github.com/red-hat-storage/cephci/blob/master/suites/reef/upgrades/tier-2-upgrade-rhcs-5x-to-7x-standby-mgr.yaml#L27

Reply:
@vamahaja this code is doing it - self.cluster.rhcs_version = _rhcs_version or rhbuild

Reply:
While Cluster is getting created as part of test_cephadm test and this value will be either _rhcs_version(if provided in suite file) or rhbuild given in CLI arguments.
_rhcs_version will take priority